src/components/Article.jsx

```diff
@@ -6,20 +6,34 @@ import Moment from 'moment';
 
 const Article = () => {
     const [article, setArticle] = useState([])
-    
-  
+    const [isLoading, setIsLoading] = useState(false);
+    const [isError, setIsError] = useState(false);
 
     const { article_id } = useParams()
 
     useEffect(() => {
+        setIsLoading(true);
```

Comment:
You can set the isLoading to be true on line 9 which then means you dont need this line🙂